### ISSUE TITLE
Tag Pair Not working 

### ISSUE THREAD

**Issue by immergetech** [bjornbjorn/SEO-Lite#36]
I'm trying to use the tag pair to get just the data from the meta description field into another location in my template. I have copied the code almost exactly from the mod file and nothing is showing. This is within a channels:entries loop but I've tested outside as well and still have the same result. I also experience the same result no matter the parameters I add in here. 

```
{exp:seo_lite:pair}
         <h3>Meta description:</h3>
           {meta_description}
    {/exp:seo_lite:pair}
```

**Comment by bjornbjorn** [comment]
Can you try with use_last_segment="yes" like this:

```       
        {exp:seo_lite use_last_segment="yes"}
            Title: {title}
            Meta keywords: {meta_keywords}
            Meta description: {meta_description}

        SEO Extra blog_body: {extra:blog_body}

            Image: {extra:blog_image}
        {/exp:seo_lite}
```

**Comment by immergetech** [comment]
Ok. Using your new beginning tag, I've got it to display the whole SEO template, which is not the desired result. I did notice that the description displays twice now.  Here is the code I am using:

`{exp:seo_lite use_last_segment="yes"} {meta_description} {/exp:seo_lite}`

Here is the results when I use the new pairs from above:

`<title>Associate Passes Bar Exam \| Timberlake, Smith, Thomas & Moses, P.C.</title>
 <meta name='description' content='Mary Baldwin and William and Mary Law School graduate Brittany Shipley passes the Virginia Bar Exam. This is exciting news for Brittany and our law firm.' />
 <link rel='canonical' href='https://www.timberlakesmith.com/news/detail/associate-brittany-shipley-passes-the-virginia-bar-exam' /> Mary Baldwin and William and Mary Law School graduate Brittany Shipley passes the Virginia Bar Exam. This is exciting news for Brittany and our law firm.`

**Comment by bjornbjorn** [comment]
Hi, it shouldn't do that -- what version of EE are you on? Also, can you pastebin the entire template?

**Comment by immergetech** [comment]
Interesting. I'm on EE 4.3.8 and I'm using SEO Lite 1.5.6. I've attached the template in this post.
[detail.txt](https://github.com/bjornbjorn/SEO-Lite/files/2862261/detail.txt)

**Comment by bjornbjorn** [comment]
Thanks, what happens if you move the seo_lite tag just below the {/exp:channel:entries} tag - does it work then?

**Comment by immergetech** [comment]
Hi bjorn,

Sorry for the late response. I've moved right below and right after the {/exp:channel:entries} tag. It still displays the description twice. Here is what it displays:

`<title>Associate Passes Bar Exam \| Timberlake, Smith, Thomas & Moses, P.C.</title>
 <meta name='description' content='Mary Baldwin and William and Mary Law School graduate Brittany Shipley passes the Virginia Bar Exam. This is exciting news for Brittany and our law firm.' />
 <link rel='canonical' href='https://www.timberlakesmith.com/news/detail/associate-brittany-shipley-passes-the-virginia-bar-exam' /> Mary Baldwin and William and Mary Law School graduate Brittany Shipley passes the Virginia Bar Exam. This is exciting news for Brittany and our law firm.`

**Comment by bjornbjorn** [comment]
Are you sure everything there is output by the {exp:seo_lite use_last_segment="yes"} tag? I'd output something just before and just after that tag to debug.

**Comment by immergetech** [comment]
I have tested that before and I've gotten the same results. I've tried placing the {exp:seo_lite use_last_segment="yes"} as the last thing in the template and it still outputted the same thing. I've also tried creating a blank template and place the {exp:seo_lite use_last_segment="yes"} {meta_description} {/exp:seo_lite} and i still get the same results.

**Comment by bjornbjorn** [comment]
That's really weird. Could you try one last thing? In the same template group as the template you attached above, create a test.html and only add:
```
{exp:seo_lite use_last_segment="yes"}
            Title: {title}
            Meta keywords: {meta_keywords}
            Meta description: {meta_description}
{/exp:seo_lite}
 ```

No snippets, layout, anything. Just that tag.

What does it output then?

**Comment by immergetech** [comment]
That's really close what I did before. So, I created a test.html in the same group as the news group and only inputted the following:

`{exp:seo_lite use_last_segment="yes"}
            Title: {title}
            Meta keywords: {meta_keywords}
            Meta description: {meta_description}
{/exp:seo_lite}`

It does not display any actual data from the the SEO lite. Only the text: Title, Meta keywords, Meta description. You can see it here:

https://www.timberlakesmith.com/news/test

**Comment by bjornbjorn** [comment]
ok, sorry I see now that it won't work cause you don't have an entry with url_title="test" .. but you can change to the use of entry_id and pull an entry you know has SEO Lite information.

Thinking about it this might be some strangeness because of EE / PHP version of something which makes it not work as expected when using as a tag pair like this. There is a "pair" tag, you can try to use that instead and see if that solves your issue:

{exp:seo_lite:pair entry_id="{entry_id}"}

{/exp:seo_lite:pair}

It basically does exactly the same as using it the other way but yeah, worth a try.

**Comment by immergetech** [comment]
Ok, I finally was able to get it to work on the template that I was trying to implement. The following code works for me perfectly:

`{exp:seo_lite:pair use_last_segment="yes"}{meta_description}{/exp:seo_lite:pair}`

It only pulls the meta description, which is exactly what I wanted. The only thing missing was use_last_segment="yes" in the original comment that I made. The pair parameter is required in order for this to work. Thanks for the help. I greatly appreciate it.
